rerun_cpp/src/rerun/serialized_component_batch.hpp

```diff
@@ -1,3 +1,5 @@
+#pragma once
```

Comment:
Ouch - we should lint for this. Should be pretty easy to add to `lint.py` (i.e. check that every `.hpp` contains a `#pragma once`)